core/src/organization/member-restriction/member.types.ts

```diff
@@ -0,0 +1,36 @@
+import {Field, ObjectType} from "@nestjs/graphql";
+
+import {Member, MemberRestrictionType} from "../../database";
+
+@ObjectType()
+export class MemberRestrictionEvent {
```

Comment:
Duplication of restriction values. Should reference member restriction and let that handle values/types. We don't want to have to update these twice if we ever change them.